### PR TITLE
[Merged by Bors] - chore: rename `ArithmeticFunction.card_divisors` -> `Nat.card_divisors`

### PR DESCRIPTION
**Pull request by llllvvuu**
Aesthetic preference, feel free to close if this doesn't make sense.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by eric-wieser — APPROVED** [review]

**Comment by eric-wieser** [comment]
maintainer merge

I agree that the old naming was weird, let's get a second opinion just in case.

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**
